images/dnsmasq/Makefile

```diff
@@ -29,7 +29,7 @@ export DOCKER_CLI_EXPERIMENTAL=enabled
 
 # Multiarch image
 # Uploaded: Jun 2, 2023, 12:57:53 PM
```

Comment:
I believe this line should be removed if we are to switch from a specific sha to `latest`.

Reply:
done